src/aspire/basis/polar_2d.py

```diff
@@ -59,8 +59,9 @@ def _precomp(self):
         omega0 = 2 * np.pi / (2 * self.nrad - 1)
         dtheta = 2 * np.pi / self.ntheta
 
-        freqs = np.zeros((2, self.nrad * self.ntheta))
-        for i in range(self.ntheta):
+        # only need half size of ntheta
+        freqs = np.zeros((2, self.nrad * self.ntheta//2))
```

Comment:
Spaces around `//`. Same goes for `//` and `*` below.

Reply:
Done.